### PR TITLE
Add docs for how to connect to Artsy Slack

### PR DESCRIPTION
**Pull request by dleve123**
Slack context: https://artsy.slack.com/archives/C02BC3HEJ/p1583340408157200

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,15 @@ Whenever we receive a new event, after storing the event locally, we use Phoenix
 
 [ashkan18]: https://github.com/ashkan18
 [zephraph]: https://github.com/zephraph
+
+## Artsy Slack Setup
+
+APRd is used to powerful critical alerting workflows in Artsy's Slack organization. After a recent incident where APRd lost its connection to Artsy's Slack, we surfaced the following steps to re-connect the digital assets needed to get it all working:
```
Comment (dblandin) [inline README.md]: s/powerful/power

**Comment by dleve123** [comment]
Tests are failing for unrelated reasons, just gonna merge anyways.

**Comment by ashkan18** [comment]
you should fix the tests! it's the postgres issue that has happened few other places too.

**Comment by dleve123** [comment]
Beat me to it! Appreciate it :)

**Pull request merged**